pkg/webui/oauth/views/change-password/index.js

```diff
@@ -0,0 +1,157 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { Container, Col, Row } from 'react-grid-system'
+import bind from 'autobind-decorator'
+import { defineMessages } from 'react-intl'
+import { push, replace } from 'connected-react-router'
+import { connect } from 'react-redux'
+import * as Yup from 'yup'
+
+import api from '../../api'
+import sharedMessages from '../../../lib/shared-messages'
+import { digit, uppercase, special } from '../../lib/regexp'
+
+import Button from '../../../components/button'
+import Form from '../../../components/form'
+import Input from '../../../components/input'
+import SubmitButton from '../../../components/submit-button'
+import IntlHelmet from '../../../lib/components/intl-helmet'
+import Message from '../../../lib/components/message'
+
+import style from '../create-account/create-account.styl'
+
+const m = defineMessages({
+  changePassword: 'Change Password',
+  passwordChanged: 'Password changed successfully',
+})
+
+const validationSchema = Yup.object().shape({
+  new_password: Yup.string()
+    .min(8)
```

Comment:
Password rules are config, not static.

Reply:
I think it's better to fix this in a separate issue to maintain the scope here.
@pgalic96 can you follow up with an issue here?